bin/internal/update_engine.ps1

```diff
@@ -0,0 +1,79 @@
+# Copyright 2021 Samsung Electronics Co., Ltd. All rights reserved.
+# Copyright 2014 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# ---------------------------------- NOTE ---------------------------------- #
+#
+# Please keep the logic in this file consistent with the logic in the
+# `update_engine.sh` script in the same directory to ensure that Flutter
+# continues to work across all platforms!
+#
+# -------------------------------------------------------------------------- #
+
+$ErrorActionPreference = "Stop"
+
+$progName = Split-Path -parent $MyInvocation.MyCommand.Definition
+$rootPath = (Get-Item $progName).parent.parent.FullName
+$enginePath = "$rootPath\bin\cache\artifacts\engine"
+$engineStamp = "$enginePath\engine.stamp"
+$engineVersion = (Get-Content "$rootPath\bin\internal\engine.version")
+$engineShortVersion = $engineVersion.Substring(0, 7)
+
+# Make sure that PowerShell has expected version.
+$psMajorVersionRequired = 5
+$psMajorVersionLocal = $PSVersionTable.PSVersion.Major
+if ($psMajorVersionLocal -lt $psMajorVersionRequired) {
+    Write-Host "Flutter requires PowerShell $psMajorVersionRequired.0 or newer."
+    Write-Host "See https://flutter.dev/docs/get-started/install/windows for more."
+    Write-Host "Current version is $psMajorVersionLocal."
+    exit 1
+}
+
+if ((Test-Path $engineStamp) -and ($engineVersion -eq (Get-Content $engineStamp))) {
+    return
+}
+
+$engineBaseUrl = $ENV:BASE_URL
+if (-not $engineBaseUrl) {
+    $engineBaseUrl = "https://github.com/flutter-tizen/engine/releases"
+}
+$engineZipName = "windows-x64.zip"
+$engineUrl = "$engineBaseUrl/download/$engineShortVersion/$engineZipName"
+$engineZipPath = "$enginePath\artifacts.zip"
+
+New-Item $enginePath -force -type directory | Out-Null
+
+Try {
+    Import-Module BitsTransfer
+    $ProgressPreference = 'SilentlyContinue'
```

Comment:
Why don't you remove this line? Downloading windows-x64.zip could take up to several minutes but I can't see its progress due to this. Also it seems you need to back up the original value of `$ProgressPreference` and restore after the operation if you want to set it to `'SilentlyContinue'` temporarily.

Reply:
Okay it's better to remove those lines.